icons/align-horizontal-distribute-end.svg

```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect width="6" height="14" x="4" y="5" rx="2" />
-  <rect width="6" height="10" x="14" y="7" rx="2" />
+  <path d="M10 17a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V7a2 2 0 0 1 2-2h2a2 2 0 0 1 2 2" />
```

Comment:
Why should we go for more code?

Reply:
This removes two places where elements are on top of each other, it's imo okay to have that.

Just wanted to know what everyone else is thinking about overlapping lines like we have here.